app/src/main/java/org/jellyfin/androidtv/model/itemtypes/LiftedItems.kt

```diff
@@ -2,9 +2,12 @@ package org.jellyfin.androidtv.model.itemtypes
 
 import org.jellyfin.apiclient.model.dto.BaseItemDto
 import org.jellyfin.apiclient.model.dto.BaseItemType
+import org.jellyfin.apiclient.model.querying.ItemFields
 import java.util.*
 import kotlin.properties.Delegates
 
+val FieldsRequiredForLift = arrayOf(ItemFields.DateCreated, ItemFields.MediaSources, ItemFields.MediaStreams, ItemFields.People)
```

Comment:
I'd hardcode this in the function that uses it

Reply:
We might need this in multiple places though and it may need to be updated once we implement more item types, which is why I put it locally close to where the classes are defined.

Reply:
Fair enough, can you change it to use the [proper naming convention](https://kotlinlang.org/docs/reference/coding-conventions.html#property-names)?

Reply:
Sure, will do